tests/kafkatest/tests/streams/streams_standby_replica_test.py

```diff
@@ -55,6 +50,9 @@ def test_standby_tasks_rebalance(self):
         processor_2 = StreamsStandbyTaskService(self.test_context, self.kafka, configs)
         processor_3 = StreamsStandbyTaskService(self.test_context, self.kafka, configs)
 
+        producer = self.get_producer(self.streams_source_topic, self.num_messages, repeating_keys=6)
+        producer.start()
```

Comment:
Why do we start the producer later compared to the driver it replaces?

Reply:
oversight during refactoring, I'll update this.